app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str):
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str):
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super(FrontendDeveloper, self).__init__(name)
+        self.skills += ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self):
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super(BackendDeveloper, self).__init__(name)
+        self.skills += ["Python", "SQL", "Django"]
+
+    def create_powerful_api(self):
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super(AndroidDeveloper, self).__init__(name)
+        self.skills += ["Java", "Android studio"]
+
+    def create_smooth_mobile_app(self):
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
+    def __init__(self, name: str):
+        super(FullStackDeveloper, self).__init__(name)
```

Comment:
You don't need to create `__init__` if all it does is call parent `__init__` with the same arguments.

Reply:
> You don't need to create `__init__` if all it does is call parent `__init__` with the same arguments.

fixed :)